dell/xps/17-9700/common.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, pkgs, ... }:
+
+{
+  # Boot loader
+  boot.loader.systemd-boot.enable = lib.mkDefault true;
+  boot.loader.efi.canTouchEfiVariables = lib.mkDefault true;
```

Comment:
Some people might prefer grub over systemd-boot.
Also enabling canTouchEfiVariables for every upgrade can destroy the efi flash.
Hence we should only enable it for installation.

```suggestion
```

Reply:
Okay, there is already a precedent for setting these though, e.g. in 

https://github.com/nixos/nixos-hardware/blob/b31fa05b585f1d40c2f9feecb3d269f4034979b1/dell/xps/15-9560/xps-common.nix#L6

Furthermore, these are lib.mkDefault... if people prefer something else they could easily override, right?

Reply:
Right but this option is still opinionated and we should rather rely on nixpkgs default. There is nothing in the hardware that mandates a specific bootloader. I am going to remove the bootloader from profile as well.

Reply:
See https://github.com/NixOS/nixos-hardware/pull/307